src/main/kotlin/lotto/domain/LottoPrize.kt

```diff
@@ -0,0 +1,32 @@
+package lotto.domain
+
+import lotto.domain.enums.PrizeType
+import lotto.domain.lotto.Lotto
+import lotto.domain.lotto.WinningLotto
+
+class LottoPrize(
+    userLottoList: List<Lotto>,
+    private val winningLotto: WinningLotto
+) {
+
+    private val _prizeResult: MutableMap<Int, Int> = (3..6)
+        .associateWith { 0 } as MutableMap<Int, Int>
```

Comment:
```suggestion
        .associateWith { 0 }
        .toMutableMap()
```
toMutableMap 함수를 활용해도 좋을 것 같아요